cuda.spec

```diff
@@ -59,3 +58,22 @@ ln -sf libcuda.so.%{driversversion}                                     %{i}/dri
 cp -ar %_builddir/nvidia/libnvidia-fatbinaryloader.so.%{driversversion} %{i}/drivers/
 cp -ar %_builddir/nvidia/libnvidia-ptxjitcompiler.so.%{driversversion}  %{i}/drivers/
 ln -sf libnvidia-ptxjitcompiler.so.%{driversversion}                    %{i}/drivers/libnvidia-ptxjitcompiler.so.1
+
+%post
+# let nvcc find its components when invoked from the command line
+cat > $RPM_INSTALL_PREFIX/%{pkgrel}/bin/nvcc.profile << @EOF
+
+TOP              = $CMS_INSTALL_PREFIX/%{pkgrel}
+
+NVVMIR_LIBRARY_DIR = \$(TOP)/nvvm/libdevice
```

Comment:
why not use sed or perl to replace _HERE_ with $CMS_INSTALL_PREFIX/%{pkgrel}/bin?

Reply:
because we also make other adjustments to the directory structure to skip the `targets/x86_64-linux/` part

Reply:
so can we also replace `_TARGET_DIR_`  with empty string then?
Overriding the full file means we might drop/add something in it which cuda might have updated in new version.

Reply:
Sorry for the delay - can you check #3745 ?